Source/WebInspectorUI/UserInterface/Controllers/ConsoleManager.js

```diff
@@ -43,6 +43,7 @@ WI.ConsoleManager = class ConsoleManager extends WI.Object
 
         this._snippets = new Set;
         this._restoringSnippets = false;
+        this._failedSourceMapConsoleMessages = new Set;
```

Comment:
NIT: could you add a newline above this so that it's distinct from the snippets "section"?

Reply:
Addressed in e2c3874b5b35844fbcd028d68867b71662ae988e.